index.html

```diff
@@ -550,9 +552,11 @@ <h2>
             |settings|, returning |previousResult|, and the user agent has not received <a>new
             information about the user's intent</a> since that invocation, return |previousResult|.
             </li>
-            <li>Return the instance of |name|'s [=powerful feature/extra permission data type=]
-            that matches the UA's impression of the user's intent, taking into account any
-            [=powerful feature/extra permission data constraints=] for |name|.
+            <li>Return the instance of |name|'s [=powerful feature/extra permission data type=] that
+            matches the UA's impression of the user's intent, combined with any configuration from
+            the user agent's [=administrator=], taking into account any [=powerful feature/extra
+            permission data constraints=] for |name|. If [=administrator=] configuration affected the
+            result, the [=user agent=] must inform the user.
             </li>
```

Comment:
I think this is how to integrate Chrome's [WebUsbAllowDevicesForUrls](https://chromeenterprise.google/policies/#WebUsbAllowDevicesForUrls) policy, which I believe adds to the set of devices that https://wicg.github.io/webusb/#dom-usb-getdevices returns by default. @reillyeon might correct me.

This text says to inform the user if configuration affects the result either by adding or removing things. Is that right, or should we only inform the user if admin configuration adds to the page's abilities?

Reply:
Yes, enterprise policy adds additional entries to the result of `getDevices()`. As implemented it cannot remove entries but I could see that as a possibility.